pkgs/tools/misc/tmux-sessionizer/default.nix

```diff
@@ -5,20 +5,32 @@
 , openssl
 , pkg-config
 , Security
-}:
+, testers
+, tmux-sessionizer
+}: let
 
-rustPlatform.buildRustPackage rec {
```

Comment:
When migrating from rec, a function with finalAttrs seems to be preferred over let-in, see the hello package for example: https://github.com/gepbird/nixpkgs/blob/master/pkgs/by-name/he/hello/package.nix

Reply:
`rustPlatform.buildRustPackage` doesn't take a function, so you can't use the `finalAttrs` pattern with it.